SEC/Equities/EquityInstruments.rdf

```diff
@@ -627,6 +706,18 @@
 		<skos:definition>dividend that falls under capital gains tax rates that are lower than the income tax rates on unqualified, or ordinary, dividends</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;RedeemablePreferredShare">
```

Comment:
Are any of these subclasses of PreferredShare disjoint?

Reply:
Yes, several pairs are disjoint, for example cumulative and non-cumulative, participating and non-participating, etc.